google-cloud-bigquery/test/google/cloud/bigquery/dataset_reference_test.rb

```diff
@@ -144,6 +145,38 @@
     _(table.query_udfs).must_be :empty?
     _(table).must_be :view?
     _(table).wont_be :table?
+    _(table).wont_be :materialized_view?
+  end
+
+  it "can create a materialized view" do
+    mock = Minitest::Mock.new
+    insert_view = Google::Apis::BigqueryV2::Table.new(
+      table_reference: Google::Apis::BigqueryV2::TableReference.new(
+        project_id: project, dataset_id: dataset_id, table_id: view_id
+      ),
+      materialized_view: Google::Apis::BigqueryV2::MaterializedViewDefinition.new(
+        enable_refresh: false,
+        query: query,
+        refresh_interval_ms: 3600000
```

Comment:
Include the underscores for readability.

Reply:
Updated, thanks!